architecture/additional_concepts/authorization.adoc

```diff
@@ -399,10 +405,16 @@ Admission uses the following approach to create the final security context for
 the pod:
 
 . Retrieve all SCCs available for use.
-. Generate field values for any security context setting that was not specified
+. Generate field values for security context settings that were not specified
 on the request.
 . Validate the final settings against the available constraints.
 
+For a *FSGroup* SCC strategy of `MustRunAs`, if the pod spec defines a _fsGroup_ id then that id value must equal the default fSGroup id (first id in the first range in the namespace's "openshift.io/sa.scc.supplemental-groups" annotation). Otherwise, the pod is not authorized by that SCC and the next SCC is evaluated.
```

Comment:
What is annotation based config? (link is ok). I see ids defined in namespaces and in pod specs. Is there a 3rd place where group (or user) Ids can be defined?